core/lib/dal/migrations/20240627142548_l1_batches_consensus.up.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE l1_batches_consensus (
+  l1_batch_number BIGINT PRIMARY KEY REFERENCES l1_batches (number) ON DELETE CASCADE,
+  certificate JSONB NOT NULL,
+  is_submitted BOOLEAN NOT NULL DEFAULT false,
```

Comment:
you mean submitted to l1 right? So it will be just `false` for a while - shall we maybe ditch it for now and introduce when we actually use it?

Reply:
Yes, L1 or main node - a synonym to "finalized". It starts as `false` and then turns `true` when we observe the QC on L1 or the main node API. 

No problem I'll remove it and add it as a migration in BFT-480 👍